apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.sql

```diff
@@ -0,0 +1,51 @@
+-- BLOCK get_course_instance
```

Comment:
We typically start query names with `select_`, `update_`, etc. (the usual SQL verbs).